### PR TITLE
feat(installer): add job docker file

### PR DESCRIPTION
**Pull request by Huimintai**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: build/docker/tools/job/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM python:3.6-alpine
+ENV HOME="/" \
+    OS_ARCH="amd64" \
```
Comment (leoryu) [inline build/docker/tools/job/Dockerfile:3]: is this env used?

**Review by huxiaoliang — CHANGES_REQUESTED** [review]
代码特别不清晰

File: build/docker/tools/job/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    OS_NAME="linux" \
+    APP_VERSION="1.22.13"
+
+RUN mkdir -p /tmp/kubectl/ && cd /tmp/kubectl && \
+    if [ ! -f kubectl-$APP_VERSION-1-$OS_NAME-$OS_ARCH-debian-11.tar.gz ]; then \
+      wget https://downloads.bitnami.com/files/stacksmith/kubectl-$APP_VERSION-1-$OS_NAME-$OS_ARCH-debian-11.tar.gz ; \
+    fi && \
+    echo "fbedde165a535402d3f53e589c088ffb36e861ee053f4147104be02807a93187  kubectl-$APP_VERSION-1-$OS_NAME-$OS_ARCH-debian-11.tar.gz" | sha256sum -c - && \
```
Comment (huxiaoliang) [inline build/docker/tools/job/Dockerfile:11]: 1. 换了 k8s 版本 这个 fbedde165a535402d3f53e589c088ffb36e861ee053f4147104be02807a93187   还能用吗？
2.  Run 可以写一个不？

File: build/docker/tools/job/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    OS_NAME="linux" \
+    APP_VERSION="1.22.13"
+
+RUN mkdir -p /tmp/kubectl/ && cd /tmp/kubectl && \
+    if [ ! -f kubectl-$APP_VERSION-1-$OS_NAME-$OS_ARCH-debian-11.tar.gz ]; then \
+      wget https://downloads.bitnami.com/files/stacksmith/kubectl-$APP_VERSION-1-$OS_NAME-$OS_ARCH-debian-11.tar.gz ; \
+    fi && \
+    export CHECKSUM=$(sha256sum kubectl-$APP_VERSION-1-$OS_NAME-$OS_ARCH-debian-11.tar.gz) && \
+    echo "$CHECKSUM" | sha256sum -c - && \
+    tar -zxf kubectl-$APP_VERSION-1-$OS_NAME-$OS_ARCH-debian-11.tar.gz -C /bin/ --strip-components=4 && \
```
Comment (huxiaoliang) [inline build/docker/tools/job/Dockerfile:13]: 这个 kubectl-$APP_VERSION-1-$OS_NAME-$OS_ARCH-debian-11.tar.gz 为啥最后又不删除了？

**Review by wl-chen — APPROVED** [review]

**Review by jo-hnny — APPROVED** [review]

**Comment by Huimintai** [comment]
> 

OK

**Review by huxiaoliang — APPROVED** [review]

**Pull request merged**